### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by Adholah96**
### In this project, I completed the following : 

- [ ] Removed all hardcoded items from the tasks array.

- [ ] Created a new JavaScript file for the new functionality.

- [ ] Implemented a function for adding a new task (add a new element to the array).

- [ ] Implement a function for deleting a task (remove an element from the array).

- [ ] Implemented a function for editing task descriptions.

- [ ] All changes to the To Do List should be saved in local storage.

### CODE REVIEW

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @Adholah96  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

## Highlights
✅ Professional PR
✅  Correct Github flow

## Required Changes ♻️
- [ ] Good job implementing the adding task and removing task functions. but I cant see the editing task description function. kindly consider implementing editing function as it is one of the requirements.
  -  Implement a function for editing task descriptions.
- [ ] Kindly consider giving your project a meaningful name and descriptive description under the title as it will make your readme more professional.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/app.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  });
+  input.addEventListener('keydown', (e) => {
+    if (e.keyCode === 13) {
+      e.preventDefault();
+      submit.click();
+    }
+  });
+};
+
+export { getData, populateList, addlist };
```
Comment (BenMukebo) [inline modules/app.js:133]: - [ ] The function of adding and deleting a new book is correctly implemented. 👍🏻  However, as requested by the previous reviewer, 🙃  It's one of the requirements of the project is that you implement a function for editing book descriptions. Please, address this and make sure that in addition to creating the book, we can also modify its content ✍🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Adholah96, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- Function for adding a new book is implemented. ✔️
- Function for remove a book from the array is implemented. ✔️
- GitHub flow is followed. :heavy_check_mark:
- All changes to the To-Do List  are saved in local storage. :heavy_check_mark:


 ## Required Changes ♻️
_Check the comments under the review._

You have submitted a project, that Linters check are not running in the GitHub/action
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly, It will be better to have the issue link in the contributing sections: 

```
## 🤝 Contributing

Feel free to check the [issues page](../../issues/).
```

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @Adholah96 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- No linter errors ✔️
- Good implementations ✔️

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**